src/main/java/net/mindengine/galen/utils/GalenUtils.java

```diff
@@ -380,16 +380,21 @@ public static InputStream findFileOrResourceAsStream(String filePath) throws Fil
             return new FileInputStream(file);
         }
         else {
-            if (!filePath.startsWith("/")) {
-                filePath = "/" + filePath;
+            if (!filePath.startsWith(File.separator)) {
+                filePath = File.separator + filePath;
             }
             InputStream stream = GalenUtils.class.getResourceAsStream(filePath);
             if (stream != null) {
                 return stream;
             }
             else {
-                String windowsFilePath = filePath.replace("\\", "/");
```

Comment:
And this was done in order to fix the problem introduced with the fix described in my above comment :)